src/stransport_stream.c

```diff
@@ -230,7 +230,8 @@ static void stransport_free(git_stream *stream)
 	stransport_stream *st = (stransport_stream *) stream;
 
 	git_stream_free(st->io);
-	CFRelease(st->ctx);
+	if (st->ctx)
+		CFRelease(st->ctx);
```

Comment:
When would this be `NULL`? If we fail to create the context, we error out on the constructor itself.